android/app/build.gradle

```diff
@@ -24,6 +25,10 @@ buildscript {
   // Add a parameter to force Firebase.
   ext.googleFirebaseServicesEnabled = project.hasProperty('firebase') ?: googleFirebaseServicesDefault
 
+  // Enable Android Auto for all debug and beta flavors except fdroid .
+  def androidAutoEnabledDefault = (isDebug || isBeta) && !isFdroid
```

Comment:
We can enable AA for fdroid builds. It works fine when phone has google services

Reply:
I disabled intentionally just to have two distinct builds in debug mode. Some people build fdroidDebug, some people build googleDebug.